search/intermediate_result.cpp

```diff
@@ -295,29 +295,30 @@ void FillDetails(FeatureType & ft, Result::Details & details)
     stars.append(osm::MapObject::kStarSymbol);
 
   auto const cuisines = feature::GetLocalizedCuisines(typesHolder);
-  auto const cuisine = strings::JoinStrings(cuisines, osm::MapObject::kFieldsSeparator);
+  auto const cuisine = strings::JoinStrings(cuisines, feature::kFieldsSeparator);
```

Comment:
nit: I suppose we can remove ```#include "indexer/map_object.hpp"``` here.

Reply:
Can't be removed. it is used in other lines